src/oscontainer.py

```diff
@@ -195,6 +196,19 @@ def oscontainer_build(containers_storage, tmpdir, src, ref, image_name_and_tag,
                   '-l', OSCONTAINER_COMMIT_LABEL + '=' + rev]
         if ostree_version is not None:
             config += ['-l', 'version=' + ostree_version]
+
+        with open('builds/builds.json') as fb:
+            builds = json.load(fb)['builds']
+        latest_build = builds[0]['id']
+        arch = cmdlib.get_basearch()
+        metapath = f"builds/{latest_build}/{arch}/meta.json"
+        with open(metapath) as f:
+            meta = json.load(f)
+        rhcos_commit = meta['coreos-assembler.container-config-git']['commit']
+        cosa_commit = meta['coreos-assembler.container-image-git']['commit']
+        config += ['-l', 'com.coreos.coreos-assembler-commit=' + cosa_commit]
+        config += ['-l', 'com.coreos.redhat-coreos-commit=' + rhcos_commit]
```

Comment:
Take a look at using `f strings`  https://www.python.org/dev/peps/pep-0498/ 

e.g. `config += ['-l', 'com.coreos.coreos-assembler-commit=f{cosa_commit}']`

Reply:
Thanks for the feedback. Have changed to f strings.